README.md

```diff
@@ -196,6 +197,13 @@ Research papers and similar documents studying something related to Matrix.
   Matrix homeserver for special purposes which compiles as a single file with
   zero dependencies, except NodeJS. `MIT` `TypeScript`
 
+## Social Media
+
+- [Circles](https://www.kombuchaprivacy.com/circles/) - E2E encrypted social
+  networking built on Matrix. Safe, private sharing for your friends, family,
+  and community. ([Repo](https://gitlab.futo.org/circles/circles-ios))
```

Comment:
A bit tricky to find this link, as it doesn't seem to be on the website (which still links to the older GitHub)... Anyway, it's clear enough that the main developer is part of FUTO, and they just need to clean up some links to make the current state more obvious.

Reply:
Yeah, I hesitated to include it when I saw the inactive GitHub repo. I'm going to file an issue to see if they'd want to update their website